test/unit/gateways/litle_test.rb

```diff
@@ -258,6 +268,14 @@ def test_add_android_pay_order_source
     end.respond_with(successful_purchase_response)
   end
 
+  def test_add_google_pay_order_source
+    stub_comms do
+      @gateway.purchase(@amount, @decrypted_google_pay)
+    end.check_request do |_endpoint, data, _headers|
```

Comment:
💬  Comment:

Maybe is good place to have a `check_request(skip_response: true) `and avoid the uneeded response check